bot/src/handler.rs

```diff
@@ -73,6 +73,12 @@ where
 
     #[instrument(skip_all)]
     async fn message(&self, ctx: Context, new_message: Message) {
+        for sticker in &new_message.sticker_items {
+            if sticker.format_type == StickerFormatType::Lottie {
+                new_message.reply_mention(&ctx, "hurensohn").await;
```

Comment:
Please keep localization in mind.

```suggestion
                new_message.reply_mention(&ctx, "fuck off").await;
```

Or even better, don't send a message at all:

```suggestion

```

Reply:
We should probably dm the user, explaining what happened (or send the insult there)

Reply:
Good idea. Maybe we can use discord's new timeout feature to give the user a timeout (e.g. 1 minute) as well.

Reply:
that seems like a good idea... Would you like to do that?

Reply:
Nö

Reply:
Das wird Arbeitsrechtliche Konsequenzen nach sich ziehen!